### ISSUE TITLE
Focus image is not a square

### ISSUE THREAD

**Issue by tiendq** [Mijick/Camera#69]
I tried a custom focus image as instructed in #53, it worked except the focus is not a square, I think there is a bug here though I looked into code and saw frame size are initialized with same value.

My code

```swift
@State var manager: CameraManager = .init(
    focusImage: UIImage(systemName: "square.dashed"),
    focusImageColor: .yellow,
    focusImageSize: 90
  )
```

And the result

![IMG_5296](https://github.com/user-attachments/assets/ada9b71f-8945-4458-aa2d-d910039da6c3)

**Comment by FulcrumOne** [comment]
Hey @tiendq,

Could you please check the `patch-3.0.0` branch and let me know if the problem occurs there as well?

Thanks

**Comment by tiendq** [comment]
> Hey @tiendq,
> 
> Could you please check the `patch-3.0.0` branch and let me know if the problem occurs there as well?
> 
> Thanks

I could change dependency to `patch-3.0.0` but I could not make it work, I see you haved made changed in names e.g. repo name and module name. It looks module name changed to `MijickCamera` but I got error `No such module 'MijickCamera'`.

**Comment by FulcrumOne** [comment]
@tiendq, there are actually a lot of changes to the public API, so this error message is random as Xcode likes to print. I will prepare a migration guide soon (hopefully this weekend, but no promises as I have a lot of other work to do).

Have a nice day,
Tomasz K.

**Comment by tiendq** [comment]
> @tiendq, there are actually a lot of changes to the public API, so this error message is random as Xcode likes to print. I will prepare a migration guide soon (hopefully this weekend, but no promises as I have a lot of other work to do).
> 
> Have a nice day, Tomasz K.

Sorry but it was not fixed in the latest release `v3.0.0`, I tested with following code and the result is same (as image posted above).

```
MCamera()
        .setCameraOutputType(.photo)
        .setAudioAvailability(false)
        .setResolution(.photo)
        .setCameraHDRMode(.off)
        .setFocusImage(UIImage(systemName: "square.dashed")!)
        .setFocusImageColor(.yellow)
        .setFocusImageSize(90)
```
